examples/colab/object_detection.ipynb

```diff
@@ -80,20 +99,18 @@
     },
     {
       "cell_type": "code",
-      "execution_count": 0,
       "metadata": {
-        "cellView": "form",
-        "colab": {},
+        "cellView": "both",
         "colab_type": "code",
-        "id": "6cPY9Ou4sWs_"
+        "id": "6cPY9Ou4sWs_",
+        "colab": {}
       },
-      "outputs": [],
       "source": [
         "#@title Imports and function definitions\n",
         "\n",
-        "# Currently %tensorflow_version 2.x installs beta1, which doesn't work here.\n",
-        "# %tensorflow_version can likely be used after 2.0rc0  \n",
-        "!pip install tf-nightly-gpu-2.0-preview\n",
+        "# Now stable version tensorflow 2.1.0 is there\n",
```

Comment:
Could you please change this line to:

# Runs with stable version tensorflow 2.1.0.

Reply:
I have now changed the line. Thanks for reviewing.